packages/design-system/README.md

```diff
@@ -0,0 +1,19 @@
+This is a provional name! We have to find our own design system name!
+
+# Naming
+## Requirements
+
+- Easy to call ```import {Button} from "@jds";```
```

Comment:
true! I'd opt for `jds` in this case. I think is simpler and easier to call from the code. I like that version instead of typing `@ferreiro/jds` any time.